app/models/comment.rb

```diff
@@ -1,6 +1,4 @@
 class Comment < ApplicationRecord
-  belongs_to :project
```

Comment:
You can still have it using `through` option:
```ruby
belongs_to :project, through: :task
```

Reply:
why this better than just ```belongs_to :project```

Reply:
this option don't have in my rails. He just throws me a mistake ```Unknown key: :through```

Reply:
Why do we need to say the model of comments belongs to project, when they belong only to task, I don't understand?

Reply:
ok